server/athenian/api/preloading/cache.py

```diff
@@ -258,11 +262,12 @@ async def _preload_all(self, debug_memory, **dbs: Dict[str, DatabaseLike]):
         self._log.info("Preloading MemoryCache")
         debug_mode = self._debug_mode if debug_memory is None else debug_memory
         tasks = []
+        enabled_accounts = await self._get_enabled_accounts(dbs["sdb"])
```

Comment:
We will have to restart API servers when we edit sdb so that they catch up with the edits. Let's re-fetch the enabled accounts on each refresh.